libs/user/store/src/lib/store/user.actions.ts

```diff
@@ -20,10 +21,11 @@ export const userActions = createActionGroup({
       message?: string;
     }>(),
     emailLoginFailure: props<{ error: HttpErrorResponse }>(),
-    creatJWTToken: emptyProps(),
-    createJWTTokenSuccess: props<{ token: string }>(),
-    createJWTTokenFailure: props<{ error: HttpErrorResponse }>(),
     logout: emptyProps(),
     getCurrentUserSuccess: props<{ user: User }>(),
+    SendEmailVerificationEmail: emptyProps(),
```

Comment:
@santoshyadavdev is UpperCamelCase recommended? We should consider this in other action names.

Reply:
My bad, changed.